alembic/versions/2024_11_11_05ffb5e13d7b_remove_order_workflow.py

```diff
@@ -0,0 +1,51 @@
+"""remove_order_workflow
+
+Revision ID: 05ffb5e13d7b
+Revises: 3503b53a9bc3
+Create Date: 2024-11-11 09:35:33.020098
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "05ffb5e13d7b"
+down_revision = "3503b53a9bc3"
+branch_labels = None
+depends_on = None
+workflows = [
+    "balsamic",
+    "balsamic-pon",
+    "balsamic-qc",
+    "balsamic-umi",
+    "demultiplex",
+    "raw-data",
+    "fluffy",
+    "microsalt",
+    "mip-dna",
+    "mip-rna",
+    "mutant",
+    "raredisease",
+    "rnafusion",
+    "rsync",
+    "spring",
+    "taxprofiler",
+    "sars-cov-2",
```

Comment:
```suggestion
    "sars-cov-2",
```
Was that never removed as an option?

Reply:
Will fix it